### PR TITLE
Fastest version of cloneDeep

### PR DESCRIPTION
**Pull request by kachurun**
Not sure that cloneDeep can strongly affect the compilation time, but JSON.Stringify is quite slow and the proposed version somewhere 2 times faster. But it is not always possible. 
If it doesn't make sense, just cancel this PR.

### CODE REVIEW

**Comment by GianlucaGuarini** [comment]
Could you please point me to a bench demonstrating it? I am curious to know more about it

**Comment by kachurun** [comment]
Yes, I'm in curious too. I am trying make true bench, but all of this "jsbench" sites are sucks.

First thing what I want to know, what is faster - Parse json which has same content-length with a lot keys, or with large values, that's answer:
https://runkit.com/kachurun/611eca1e910c2400147a5499

Much easier parse large content with small count of keys. Hard to parse deep objects. 
It can help us to understand why JSON may be faster or slower in different situations.

I'll come back later.

**Comment by kachurun** [comment]
Same example with lodash.cloneDeep:
https://runkit.com/kachurun/611ed30f29538900142800c1

**Comment by kachurun** [comment]
That were fun, but I think there is no points to increase perfomance, will close PR.

**Comment by GianlucaGuarini** [comment]
@kachurun thank you for your tests it has been really interesting reading them
